piwheels/__init__.py

```diff
@@ -74,7 +74,7 @@
 __url__          = 'https://www.piwheels.org/'
 __platforms__    = 'ALL'
 
-__requires__ = ['configargparse', 'pyzmq', 'voluptuous', 'cbor2']
+__requires__ = ['configargparse', 'pyzmq', 'voluptuous', 'cbor2', 'simplejson']
```

Comment:
Isn't simplejson strictly a master-only requirement? For that matter is it actually a requirement or a recommendation now? If it's actually a requirement we can just replace all `import json` lines with `import simplejson as json`